src/dev/register_git_hook/register_git_hook.js

```diff
@@ -111,7 +111,15 @@ function getKbnPrecommitGitHookScript(rootPath, nodeHome, platform) {
     exit 1
   }
 
-  node scripts/precommit_hook || {
+  execute_local_precommit_hook() {
+    PRECOMMIT_FILE=".pre-commit"
```

Comment:
Could be in `config/.pre-commit` as well perhaps?

Reply:
How about `.precommit-hook`, we generally don't treat `pre-commit` as two words.

Reply:
alright, followed git's convention but I always forgot about the dash anyway 🤓